config/routes.rb

```diff
@@ -1875,6 +1878,7 @@
     post "caliper/:token", controller: :lti_api, action: :caliper_service, as: "lti_caliper"
     post "logout_service/:token", controller: :lti_api, action: :logout_service, as: "lti_logout_service"
     post "turnitin/outcomes_placement/:tool_id", controller: :lti_api, action: :turnitin_outcomes_placement, as: "lti_turnitin_outcomes_placement"
+    post "vericite/outcomes_placement/:tool_id", controller: :lti_api, action: :vericite_outcomes_placement, as: "lti_vericite_outcomes_placement"
```

Comment:
This is for turnitin the LTI version, not the plugin.  This is unneeded for an non-LTI backend plugin like this.